### PR TITLE
Fix bad manifests and remove parsing test exclusions

### PR DESCRIPTION
**Pull request by sjmiles**

### CODE REVIEW

File: particles/Demo/Demo.recipes [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import 'DemoData.manifest'
+import 'TVMazeDemo.recipes'
+
 //import 'RestaurantsDemo.recipes'
 //import 'ProductsDemo.recipe'
-import 'TVMazeDemo.recipes'
```
Comment (Cypher1) [inline particles/Demo/Demo.recipes]: Does this change make a difference? I haven't touched the comment parser yet but can take a look (as to me this looks like it should work).

File: src/tests/particles/particles-test.ts [hunk 2]
```diff
@@ -4,14 +4,8 @@ import glob from 'glob';
 import { Loader } from '../../runtime/loader.js';
 
 // Skip these files, they do not compile successfully.
-// TODO: Figure out why.
 const FILES_TO_SKIP = new Set([
```
Comment (Cypher1) [inline src/tests/particles/particles-test.ts]: Can we remove this variable and comment now?

File: particles/Demo/Demo.recipes (hunk 1 above)
Comment (sjmiles) [reply particles/Demo/Demo.recipes]: There is no semantic change here.

File: src/tests/particles/particles-test.ts (hunk 2 above)
Comment (sjmiles) [reply src/tests/particles/particles-test.ts]: I thought to leave that decision to @csilvestrini, but I can amend the PR as needed.

Comment (sjmiles) [reply src/tests/particles/particles-test.ts]: No problemo.

**Pull request merged**